src/types/tag.ts

```diff
@@ -1,3 +1,3 @@
 export type HTMLTag = keyof HTMLElementTagNameMap;
 
-export type ListTag = 'ol'|'ul';
+export type ListHTMLTag = 'ol'|'ul';
```

Comment:
Doesn't it sound better as `HTMLListTag`? Just an idea :D 🙈

Reply:
Yes, you're right, it sounds better 👍 😄